products/compute/terraform.yaml

```diff
@@ -15,21 +15,42 @@
 overrides: !ruby/object:Provider::ResourceOverrides
   Address: !ruby/object:Provider::Terraform::ResourceOverride
     exclude: true
+  BackendBucket: !ruby/object:Provider::Terraform::ResourceOverride
+    id_format: "{{name}}"
+    documentation: !ruby/object:Provider::Terraform::Documentation
+      examples: |
+        ```hcl
+        resource "google_compute_backend_bucket" "image_backend" {
+          name        = "image-backend-bucket"
+          description = "Contains beautiful images"
+          bucket_name = "${google_storage_bucket.image_bucket.name}"
+          enable_cdn  = true
+        }
+
+        resource "google_storage_bucket" "image_bucket" {
+          name     = "image-store-bucket"
+          location = "EU"
+        }
+        ```
   BackendService: !ruby/object:Provider::Terraform::ResourceOverride
     exclude: true
   Disk: !ruby/object:Provider::Terraform::ResourceOverride
+    id_format: "{{name}}"
```

Comment:
Make `{{name}}` the default value for `id_format`.

Reply:
Done